libs/usb-drive/src/usb_drive.ts

```diff
@@ -113,8 +161,16 @@ export function detectUsbDrive(): UsbDrive {
   // Store mounting state so we don't try to mount the drive multiple times.
   let isMounting = false;
 
+  // The block device info blips during formatting, so we need to track the
+  // state and ignore the OS-provided device info during formatting.
+  let isFormatting = false;
+
   return {
     async status(): Promise<UsbDriveStatus> {
+      if (isFormatting) {
+        return { status: 'ejected' };
+      }
+
```

Comment:
Due to the aforementioned blips, I think that it would be thrashing consumers to actually base status on the `lsblk` received info. So I added this escape hatch, as existed for `useUsbDrive`. Re-using `ejected` rather than a new status is preferable because it involves fewer statuses that consumers have to account for, and `ejected` represents the case pretty well.